bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -40,12 +41,11 @@ zeroize = { workspace = true }
 
 
 [target.'cfg(target_os = "android")'.dependencies]
-tracing = { version = "0.1.29", default-features = false, features = ["log"] }
-android_logger = "0.11"
 log-panics = { version = "2", features = ["with-backtrace"]}
```

Comment:
Still android-specific? Do you want to change that later when replacing it?

Reply:
Yep.